src/main/java/frc/robot/Input.java

```diff
@@ -38,4 +38,13 @@ public void periodic() {
    *  return controllerVariableName.someGetMethod();
    * }
    */
+  public static boolean getA(){
```

Comment:
Instead of calling this getA, please call it getIntake() or something so everyone knows what the A button does. make sure to update this in intake command also.